workflow-bot-app/src/main/java/com/symphony/bdk/workflow/client/ApiClientFactory.java

```diff
@@ -0,0 +1,37 @@
+package com.symphony.bdk.workflow.client;
+
+import com.symphony.bdk.core.config.model.BdkConfig;
+import com.symphony.bdk.http.api.ApiClient;
+import com.symphony.bdk.http.api.ApiClientBuilder;
+import com.symphony.bdk.http.jersey2.ApiClientBuilderProviderJersey2;
+
+import org.apache.commons.lang3.StringUtils;
+
+public class ApiClientFactory {
```

Comment:
Don't we need to support truststore configuration as well? (cf. https://github.com/finos/symphony-bdk-java/blob/main/symphony-bdk-core/src/main/java/com/symphony/bdk/core/client/ApiClientFactory.java)